src/lib/apiService.ts

```diff
@@ -152,7 +182,8 @@ const getAccountTransactions = async (accountId: string, startDate?: Moment, end
     if (response && response.status === 200 && response.data.data) {
       return response.data.data;
     }
-    return [];
+    const transactions = await Array.fromAsync(pageThroughResource(http, `/accounts/${accountId}/transactions${queryString}`));
+    return transactions;
```

Comment:
Transactions needs to be cast here:
`return transactions as FF3Wrapper<FF3Transaction>[];`